setup.py

```diff
@@ -6,7 +6,7 @@
 
 setuptools.setup(
     name="sanpy",
-    version="0.7.15",
+    version="0.7.17",
```

Comment:
why 2 versions?

Reply:
I was looking at the PR and the version was 0.7.16, it seems that it is actually 0.7.17 at the moment. I bumped it to 0.7.18 now